modules/go-settings-for-environment-labels-and-annotations.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * /gitops/health-information-for-resources-deployment.adoc
+
+:_content-type: REFERENCE
+[id='go-settings-for-environment-labels-and-annotations_{context}']             
+= Settings for environment labels and annotations
+
+This section provides reference settings for environment labels and annotations required to display an environment application in the *Environments* page, in the *Developer* perspective of the {product-title} web console.
+
+[discrete]
+== Environment labels
+
+The environment application manifest must contain `labels.openshift.gitops/environment` and `destination.namespace` fields. You must set identical values for the `<environment-name>` variable and the name of the environment application manifest.
+
+.Specification of the environment application manifest
+[source,yaml]
+----
+spec:
+  labels:
+    openshift.gitops/environment: <environment-name>
+  destination:
+    namespace: <environment-name>
+...
+----  
+
+.Example of an environment application manifest
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: dev-env <1>
```

Comment:
Would these also not be in brackets? 

```suggestion
  name: <dev_env> <1>
```

Maybe it should just be dev-env. I'm not sure based on the description below. If it is the same as <environment_name>, why not just use the value "<environment_name>" ?

Reply:
This should stay as dev-env , since this is an example assuming that the user named the environment "dev-env".

Reply:
@masauso-lungu Yes. retain as is.